fdbserver/DataDistribution.actor.cpp

```diff
@@ -528,8 +531,8 @@ struct DataDistributor : NonCopyable, ReferenceCounted<DataDistributor> {
 	// usage if it turns out to be a problem.
 	Future<Void> resumeRelocations() {
 		ASSERT(shardsAffectedByTeamFailure); // has to be allocated
-		return runAfter(resumeFromShards(Reference<DataDistributor>::addRef(this), g_network->isSimulated()),
-		                resumeFromDataMoves(Reference<DataDistributor>::addRef(this)));
+		Future<Void> shardsReady = resumeFromShards(Reference<DataDistributor>::addRef(this), g_network->isSimulated());
+		return resumeFromDataMoves(Reference<DataDistributor>::addRef(this), shardsReady);
```

Comment:
I guess `runAfter` already guarantees the `resumeFromDataMoves` run after `wait(resumeFromShards)` return. Did you encounter a bug with it?